orientdb/src/test/java/org/hibernate/datastore/ogm/orientdb/OrientDBSimpleTest.java

```diff
@@ -0,0 +1,280 @@
+		LOG.log( Level.INFO, "start" );
+		// MemoryDBUtil.prepareDb("remote:localhost/pizza");
+		MemoryDBUtil.createDbFactory( "memory:test" );
+		BasicConfigurator.configure();
+		emf = Persistence.createEntityManagerFactory( "hibernateOgmJpaUnit" );
+		em = emf.createEntityManager();
+		em.setFlushMode( FlushModeType.COMMIT );
+
+	}
+
+	@AfterClass
+	public static void tearDownClass() {
+		LOG.log( Level.INFO, "start" );
+		if ( em != null ) {
+			em.close();
+			emf.close();
+
+		}
+		MemoryDBUtil.getOrientGraphFactory().close();
+	}
+
+	@Before
+	public void setUp() {
+	}
+
+	@After
+	public void tearDown() {
+		em.clear();
+	}
+
+	@Test
+	public void test1InsertNewCustomer() throws Exception {
+		LOG.log( Level.INFO, "start" );
+		try {
+			em.getTransaction().begin();
+			Customer newCustomer = new Customer();
+			newCustomer.setName( "test" );
+			LOG.log( Level.INFO, "New Customer ready for  persit" );
+			em.persist( newCustomer );
+			em.flush();
+			Query query = em.createNativeQuery( "select from Customer where name=:name", Customer.class );
+			query.setParameter( "name", "test" );
+			List<Customer> customers = query.getResultList();
+			LOG.log( Level.INFO, "customers.size(): {0}", customers.size() );
+			assertFalse( "Customers must be", customers.isEmpty() );
+			Customer testCustomer = customers.get( 0 );
+			assertNotNull( "Customer with 'test' must be saved!", testCustomer );
+			em.getTransaction().commit();
+		}
+		catch (Exception e) {
+			LOG.log( Level.SEVERE, "Error", e );
+			em.getTransaction().rollback();
+			throw e;
+		}
+	}
+
+	@Test
+	public void test1InsertNewPizza() throws Exception {
+		LOG.log( Level.INFO, "start" );
+		try {
+			em.getTransaction().begin();
+			Pizza newPizza = new Pizza();
+			newPizza.setName( "Marinero" );
+			LOG.log( Level.INFO, "New Pizza ready for  persit" );
+			em.persist( newPizza );
+			em.flush();
+			Query query = em.createNativeQuery( "select from Pizza where name=:name", Pizza.class );
+			query.setParameter( "name", "Marinero" );
+			List<Pizza> pizzaList = query.getResultList();
+			LOG.log( Level.INFO, "pizzaList.size(): {0}", pizzaList.size() );
+			assertFalse( "pizzaList must be not empty!", pizzaList.isEmpty() );
+			Pizza testPizza = pizzaList.get( 0 );
+			assertNotNull( "Pizza with 'Marinero' must be saved!", testPizza );
+			em.getTransaction().commit();
+		}
+		catch (Exception e) {
+			LOG.log( Level.SEVERE, "Error", e );
+			em.getTransaction().rollback();
+			throw e;
+		}
+	}
+
+	@Test
+	public void test2FindCustomer() {
+		LOG.log( Level.INFO, "start" );
+		try {
+			em.getTransaction().begin();
+			Customer customer = em.find( Customer.class, Long.valueOf( 2L ) );
+			em.refresh( customer );
+			LOG.log( Level.INFO, "read entity properties:" );
+			LOG.log( Level.INFO, "customer.getbKey():{0}", customer.getbKey() );
+			LOG.log( Level.INFO, "customer.getName(): {0}", customer.getName() );
+			LOG.log( Level.INFO, "customer.getRid(): {0}", customer.getRid() );
```

Comment:
Why all this log?
I think it would help if you have a look at the existing tests we have in core or in the other dialects